src/Packages/UnityWebBrowser/Runtime/Core/RawImageUwbClientInputHandler.cs

```diff
@@ -105,26 +119,27 @@ private IEnumerator KeyboardAndMouseHandler()
                     float scroll = inputHandler.GetScroll();
                     scroll *= browserClient.BrowserTexture.height;
 
-                    if (scroll != 0)
+                    if (!disableMouse && scroll != 0)
                         browserClient.SendMouseScroll(pos, (int) scroll);
                     
                     //Input
                     WindowsKey[] keysDown = inputHandler.GetDownKeys();
                     WindowsKey[] keysUp = inputHandler.GetUpKeys();
                     string inputBuffer = inputHandler.GetFrameInputBuffer();
                     
-                    if(keysDown.Length > 0 || keysUp.Length > 0 || inputBuffer.Length > 0)
+                    if(!disableKeyboard && (keysDown.Length > 0 || keysUp.Length > 0 || inputBuffer.Length > 0))
                         browserClient.SendKeyboardControls(keysDown, keysUp, inputBuffer);
```

Comment:
Same here, just do one check for `!disableKeyboard` with all logic inside of it.